### PR TITLE
added CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by rpandox**
### Summary

This PR updates the `CONTRIBUTING.md` file to provide clear instructions on forking, cloning, committing, and creating pull requests.

### Changes Made

- Added step-by-step guidelines for contributing.
- Enhanced the pull request section for clearer descriptions.

These improvements aim to streamline the contribution process for all users.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+```
+
+### Customization Tips:
+- Replace `[Repository Name]` with the actual name of your repository.
+- Update URLs for your specific GitHub repository.
+- You can add more sections if necessary, such as testing guidelines or specific coding standards.
+- Make sure to link to your project's Code of Conduct if applicable.
+
+Feel free to adjust any part of this template to better fit your project's needs!
+```
```
Comment (gaurovgiri) [inline CONTRIBUTING.md:110]: Certainly!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+4. [Making Changes](#making-changes)
+5. [Committing Your Changes](#committing-your-changes)
+6. [Pushing Your Changes](#pushing-your-changes)
+7. [Creating a Pull Request (PR)](#creating-a-pull-request-pr)
+8. [Code of Conduct](#code-of-conduct)
+9. [Additional Resources](#additional-resources)
+
+## Forking the Repository
+
+1. Navigate to the [repository](https://github.com/your_username/repository_name).
```
Comment (gaurovgiri) [inline CONTRIBUTING.md:19]: add computer club username and repository

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
Need some changes, make the computer club's tailored contributing markdown file

**Comment by rpandox** [comment]
Certainly! : removed the part of the git
add computer club username and repository

**Review by gaurovgiri — APPROVED** [review]
For now this seems okay, might need to include conventions and code of conducts and also screenshots later.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Contributing to The Website
```
Comment (aayush105) [inline CONTRIBUTING.md:1]: the website?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+3. This creates a copy of the repository in your GitHub account.
+
+## Cloning the Repository
+
+1. Go to your forked repository on GitHub.
+2. Click on the **Code** button and copy the URL (either HTTPS or SSH).
+3. Open your terminal (or Git Bash) and run the following command:
+
+   ```bash
+   git clone https://github.com/computerclubkec/computerclubkec.github.io.git
```
Comment (aayush105) [inline CONTRIBUTING.md:30]:
```suggestion
   git clone https://github.com/your-username/computerclubkec.github.io.git
```

**Review by aayush105 — CHANGES_REQUESTED** [review]

**Comment by rpandox** [comment]
Added according to the review

**Review by gaurovgiri — APPROVED** [review]
LMAO! I completely missed those. Looks good for merge

**Review by aayush105 — APPROVED** [review]
Good job, LGTM !!!

**Pull request merged**